src/script/media/MediaConstraintsHandler.js

```diff
@@ -24,13 +24,21 @@ export default class MediaConstraintsHandler {
     return {
       DEFAULT_DEVICE_ID: 'default',
       SCREEN_CONSTRAINTS: {
-        CHROME: {
+        DESKTOP_CAPTURER: {
           mandatory: {
             chromeMediaSource: 'desktop',
             maxHeight: 1080,
             minHeight: 1080,
           },
         },
+        DISPLAY_MEDIA: {
+          video: {
+            height: {
+              ideal: 1080,
+              max: 1080,
+            },
+          },
+        },
         FIREFOX: {
```

Comment:
Let's maybe rename this to `GET_USER_MEDIA` to be consistent with the other two, WDYT?

Reply:
Yeah, that would be more consistent then.